core/integration/module_test.go

```diff
@@ -479,6 +479,20 @@ func TestModuleInitLICENSE(t *testing.T) {
 		require.Contains(t, content, "Apache License, Version 2.0")
 	})
 
+	t.Run("do not boostrap LICENSE file if license is set empty", func(t *testing.T) {
+		t.Parallel()
+
+		c, ctx := connect(t)
+
+		modGen := c.Container().From(golangImage).
+			WithMountedFile(testCLIBinPath, daggerCliFile(t, c)).
+			WithWorkdir("/work").
+			With(daggerExec("init", "--name=empty-license", "--sdk=go", "--license="))
+
+		_, err := modGen.File("LICENSE").Contents(ctx)
+		require.Error(t, err)
```

Comment:
I suggest looking for the file with perhaps `Entries`. Checking for any error on `Contents` can lead to false positives.